### ISSUE TITLE
Update README.md, remove LaTeX references, include AsciiDoc information

### ISSUE THREAD

**Issue by cstrotm** [BetterCrypto/Applied-Crypto-Hardening#334]
The current readme explain how to contribute using LaTeX. These text should be changed to now address Asciidoc

**Comment by sebix** [comment]
Definitely, we need instructions for compiling the asciidoc document. Getting lots of errors here and I guess this is because of not doing the compilation correctly.

@patrickbenkoetter
